SEDaily-IOS/CustomTabViewController.swift

```diff
@@ -145,4 +154,38 @@ extension CustomTabViewController {
     func presentLeftSideMenu() {
         present(SideMenuManager.menuLeftNavigationController!, animated: true, completion: nil)
     }
+
+    fileprivate func askForReview() {
+        let popup = PopupDialog(title: "Hello! 👋",
+                                message: "Enjoying the SEDaily app?",
```

Comment:
Hopefully we can get the localization changes in soon so that these strings aren't hardcoded.